File: backend/app/src/models/locations/City.py
"use strict";
from ..blueprints.Settlement import Settlement


class City(Settlement):
    def __init__(self, name, gold, food, wood, iron, buildings = ..., population = ..., settlement_type="city"):
        super().__init__(name=name, gold=gold, food=food, wood=wood, iron=iron, buildings=buildings, population=population, settlement_type=settlement_type)

File: backend_refactoring/CSharpApi/Models/BluePrints/Locations/City.cs

using CSharpApi.Models.BluePrints.Beings;

namespace CSharpApi.Models.BluePrints.Locations
{
    public class City : Settlement
    {
        public City(string settlementName, float gold, float food, float wood, float iron, List<BuildingState> buildingStates = null!, List<Population> population = null!, List<IBuildingList> buildingLists = null!, string settlementType = "city") 
            : base(settlementName, settlementType, gold, food, wood, iron, buildingStates ?? [], population ?? [], buildingLists ?? [])
        {
        }
    }
}
